src/pages/HomePage/ListOfHouses/ListOfHouses.jsx

```diff
@@ -59,7 +59,7 @@ function ListOfHouses() {
         <Box component="div" className={styles.houseElement}>
           <h4>420 Baker St, London</h4>
           <p className={styles.price}>299,999£</p>
-          <img src="\src\assets\images\House.png" alt="House" />
+          <img src="/src/assets/images/House.png" alt="House" />
```

Comment:
I think this `src` could be removed along with the corresponding asset as there will be either an actual photo or a placeholder.

Reply:
Patryk will do this in his own branch. Changes above are just my attempts to load images to netlify.